### DISCUSSION TITLE
Woensdag 02-10-2024

### DISCUSSION THREAD

**Discussion by Annevd** [rutgerkock/buurtcampus-oost#3] [Stand-ups]
- Hoe gaat het met je?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by rutgerkock** [comment]
**Hoe gaat het met je?**
Ben bezig met Harry data meegeven aan de hand van specifiek planten en het weer buiten. Met het weer buiten is gelukt, en aan de hand van data van planten is ook gelukt. Alleen is momenteel nog de data van het weer nog dynamisch, die van planten niet.

**Wat wil je deze week bereiken (aan werken)?**
Ik wil zorgen dat alle tekst uit Hygraph komt en dat de planten ook in hypgraph data meekrijgen, aan de hand waarvan ik dan tekst/svg's kan meegeven op de pagina van de plant. Ook moet het een component worden.

**Waar loop je tegen aan?**
Ben nog niet begonnen met Hygraph, zal moeten uitvogelen hoe dit werkt.

**Comment by Annevd** [comment]
**Hoe gaat het met je?**

Afgelopen paar dagen ben ik bezig geweest met het analyseren van Hygraph, en hier nieuwe content aan toe te voegen. Ik ben op dit moment bezig met ontwerpen en bouwen van de accessibility tools button. Hiermee bedoel ik nu alleen de functionaliteit om het menu te openen en te sluiten. Nog niet de tools zelf.


**Wat wil je deze week bereiken (aan werken)?**

Ik wil deze week/voor de sprint review graag de basis voor de accessibility button af hebben, zodat ik iets van code gemaakt heb en iets heb om te laten zien aan de opdrachtgever.

**Waar loop je tegen aan?**

Omdat ik heel lang in de analysefase blijf zitten overal, heb ik nog niet veel fysieks dat ik aan de opdrachtgever kan laten zien. Ik ben veel achter de schermen bezig geweest en moet daarnaast ook nog de codebase zelf analyseren voordat ik begin met coderen.
Ik moet namelijk ook nog uitzoeken hoe het werken met componenten werkt.

**Comment by Nazneen05x** [comment]
Hoe gaat het met je?

Het gaat prima. Ik had maandag aangeven dat ik deze week de agenda-carousel wil afronden, dat gaat mij niet lukken want ik heb een probleem met het laten werken van de next en prev knoppen. 

Wat wil je deze week bereiken (aan werken)?

Het laten werken van de buttons  en PE implementeren op de buttons.

[#22](https://github.com/users/Nazneen05x/projects/14/views/1?pane=issue&itemId=80037341&issue=Nazneen05x%7CS4-S19-Bieb-In-Bloei%7C22)

[#24](https://github.com/users/Nazneen05x/projects/14/views/1?pane=issue&itemId=80037388&issue=Nazneen05x%7CS4-S19-Bieb-In-Bloei%7C24)

Waar loop je tegen aan?

Ik heb problemen met het laten werken van mijn oude javascript code voor het laten werken van de  next en prev knoppen. Ik krijg steeds een Typescript error. Ik heb typescript helemaal verwijderd uit de codebase en package verwijderd en opnieuw geinstalleerd, maar krijg nog steeds Typescript error.

**Comment by wingsvn** [comment]
**Hoe gaat het met je?**
Goed, ik heb de bieb-in-bloei website verder op toegankelijkheid getest. Er kwam hier een hoge score uit. Maar ook een aantal punten die verbetert kunnen worden. Hiervoor heb ik issues: #161 en #167 voor aangemaakt. 

**Wat wil je deze week bereiken (aan werken)?**
Anne en ik hadden besloten allebei aan de accessibility tool button te werken. Dat we een ieder de knop bouwen en vervolgens een uitkiezen. Vanuit de gekozen knop zullen we de accessibility features binnen deze knop verdelen. Deze week wil ik dus beginnen aan het bouwen van een accessibility tool button

**Waar loop je tegen aan?**
Ik moet nog onderzoeken hoe Hygraph en de huidige code in elkaar zitten.

**Comment by Lmikkers** [comment]
**Hoe gaat het met je?**
Goed, het is gelukt om Harry uit te laden, dus daar ben ik blij mee. Vond het een beetje lastig wat ik nou precies moest/kon doen, maar na de standup heb ik hier een wat duidelijker beeld van.

**Wat wil je deze week bereiken (aan werken)?**
Ik wil de footer een paar bugs daarvan oplossen en verder werken aan Harry. Ik ga samen met Rutger kijken hoe we dit kunnen kopellen aan het weer en hoe we data dynamisch kunnen maken.

**Waar loop je tegen aan?**
Ik heb nog niet met Hygraph gewerkt, hier moet ik nog onderzoek naar doen hoe het werkt. Ik wil Harry ook dynamisch maken, dus moet er data toegevoegd worden aan Hygraph. Ook werkte ik nu lokaal op mijn eigen project om te kijken of alles werkt, maar nu ga ik werken in ons project met branches, dus hoop dat dat goed gaat. :)
